updater/lib/dependabot/updater/security_update_helpers.rb

```diff
@@ -148,10 +148,10 @@ def record_security_update_dependency_not_found
 
       sig { params(lowest_non_vulnerable_version: T.nilable(String)).returns(String) }
       def earliest_fixed_version_message(lowest_non_vulnerable_version)
-        if lowest_non_vulnerable_version
+        if lowest_non_vulnerable_version && !lowest_non_vulnerable_version.empty?
```

Comment:
This shouldn't be needed since we are also shipping:
* https://github.com/dependabot/dependabot-core/pull/11949

However, not a bad idea to have defense-in-depth.